elasticsearch_collectd.py

```diff
@@ -532,10 +550,17 @@ def init_stats():
     else:
         ES_INDEX_URL = "http://" + ES_HOST + ":" + \
                        str(ES_PORT) + "/" + ",".join(ES_INDEX) + "/_stats"
+    #common thread pools for all ES versions
+    thread_pools = ['generic', 'index', 'get', 'snapshot', 'bulk', 'warmer', 'flush', 'search', 'refresh']
+
+    if ES_VERSION.startswith("2."):
+        thread_pools.extend(['suggest', 'percolate', 'management', 'listener', 'force_merge',
+                    'fetch_shard_store', 'fetch_shard_started'])
+    else:
+        thread_pools.extend(['merge', 'optimize'])
```

Comment:
I couldn't fix it easily. 
One way to address this is to specify the ES version explicitly in the config. And that's what I did for testing
Another one is to fix either our testing framework, so that ES would start before  the collectd plugin.